DBQnA/ui/react/public/index.html

```diff
@@ -0,0 +1,45 @@
+<!--
```

Comment:
Do we need this file? I think src/index.html is used. Please remove this file if not used.

Reply:
All the files in public folder is not needed? Please check and remove if not used.